README.md

```diff
@@ -55,25 +36,18 @@ Find quick installation instructions here:
 
 More editor integrations coming soon.
 
-## GitHub Code Reviews
-
-Ensure your code is always at the highest quality. Sourcery reviews and makes suggestions on your whole repo when you install and then checks all new pull requests in the future.
-
-You can get set up and running in 1 minute. Just follow this [link](https://github.com/apps/sourcery-ai/installations/new) and select your repos.
+## Feedback
 
-Or to try it out right away, ⭐ this repo, and Sourcery will review your most popular Python repo and send you a pull request. As an added bonus **@sourcery-ai-bot** will ⭐ your repo too.
-
-## Demo
-
-![Sourcery demo](sourcery-demo.gif)
+We're always trying to learn how we can make Sourcery better. Let us know any thoughts you have by emailing tim@sourcery.ai
 
 ## **Links**
 
-- [Docs](https://docs.sourcery.ai/)
```

Comment:
why remove the docs?

Reply:
They're not relevant to the current functionality. Will re-add when we decide if we're using the existing Sourcery docs or if we are looking to have the review docs live elsewhere